### PR TITLE
refactor: remove moment from AnnotationForm.tsx

### LINKED ISSUE #2057: Moment Refactor: remove moment from AnnotationForm
We need to remove `moment` from AnnotationForm.tsx

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes #2057

part of : #1844

This PR removes `moment` from `AnnotationForm.tsx`

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2058)

**Comment by hoorayimhelping** [comment]
@jrenee42 are these functions integration tested?

**Review by hoorayimhelping — APPROVED** [review]
looks good to me, but run this buy jill before merging in.

also, the semantic check was stuck, so I just modified the title then modified back to what it was and it re-ran.

File: src/annotations/components/annotationForm/AnnotationForm.tsx [hunk 1]
```diff
@@ -160,12 +159,12 @@ export const AnnotationForm: FC<Props> = (props: Props) => {
...
     if (props.getNow) {
-      now = moment(props.getNow())
+      now = new Date(props.getNow())
     } else {
-      now = moment()
+      now = new Date()
     }
 
-    return moment(timeToCheck).isAfter(now)
+    return new Date(timeToCheck).getTime() > now.getTime()
```
Comment (jrenee42) [inline src/annotations/components/annotationForm/AnnotationForm.tsx:167]: looks good; but since 'isAfter' is used twice; please put in a method!  (DRY)

**Review by jrenee42 — CHANGES_REQUESTED** [review]

File: src/annotations/components/annotationForm/AnnotationForm.tsx (hunk 1 above)
Comment (ChitlangeSahas) [reply src/annotations/components/annotationForm/AnnotationForm.tsx:167]: Got it! Will do 👍

**Comment by jrenee42** [comment]
> @jrenee42 are these functions integration tested?

they are unit-tested.  hence why 'now' is injectable for testing.

File: src/annotations/components/annotationForm/AnnotationForm.tsx (hunk 1 above)
Comment (ChitlangeSahas) [reply src/annotations/components/annotationForm/AnnotationForm.tsx]: Done ✅

**Review by jrenee42 — APPROVED** [review]

**Pull request merged**